applications/ISO20022.md

```diff
@@ -0,0 +1,185 @@
+- What need(s) does your project meet?
+	- Explore possibilities and gaps of Polkadot / Substrate support of ISO20022 standard for future improvements 
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+  - If so, how is your project different?
+  - If not, are there similar projects in related ecosystems? There are known blockchains already claim to support ISO20020 such as Ripple XRP, XLM, Algorand, etc. 
+
+## Team 
+
+### Team members
+
+- Name of team leader: Daniel Deng
+
+### Contact
+
+- **Contact Name:** Daniel Deng
+- **Contact Email:** OpenSmartContract@gmail.com
+
+### Legal Structure
+
+- **Registered Address:** No registred entity
+- **Registered Legal Entity:** No registred entity
+
+### Team's experience
+
+Daniel is the Chief Product Officer at HeroDAO where he has led blockchain based smart contract products and development on Polkodot / Kusama blockchains. Previous at Meta he managed crypto (Libra / Diem) and fiat payments and web3 initiatives. Prior to that, he was Senior Director of Product Management at Visa managing Visa's global payment product portfolios and platforms.
+
+Here's a recent[ Medium article](https://medium.com/@opensmartcontract/learn-ink-by-example-order-food-on-blockchain-a4024b2dee4a) on ink! smart contract with Rust macros implementing dApp on Dotsama chain, with [Smart Contract Deployment and Interaction Guide using Contracts-UI and Polkadot.js](https://github.com/InkSmartContract/BlockchainFoodOrder/blob/main/BlockchainFoodOrder%20Smart%20Contract%20Deployment%20and%20Interactions%20Guide.pdf).        
+
+### Team Code Repos
+- [Order Food on the Blockchain](https://github.com/InkSmartContract/BlockchainFoodOrder)
+
+Please also provide the GitHub accounts of all team members. 
+- [GitHub1](https://github.com/OpenSmartContract) 
+- [Github2](https://github.com/FPLeader) 
+
+### Team LinkedIn Profiles (if available)
+- [Profile](https://www.linkedin.com/in/dengdaniel)
+
+## Development Status 
+
+- [RFPs](https://github.com/w3f/Grants-Program/tree/master/docs/RFPs) (requests for proposal)
+- [Blockchain Food Order dApp](https://github.com/InkSmartContract/BlockchainFoodOrder/blob/main/Learn%20ink!%20by%20Example%20%E2%80%94%20Order%20Food%20on%20the%20Blockchain.pdf)
+- [Smart Contract Deployment and Interaction Guide using Contracts-UI and Polkadot.js](https://github.com/InkSmartContract/BlockchainFoodOrder/blob/main/BlockchainFoodOrder%20Smart%20Contract%20Deployment%20and%20Interactions%20Guide.pdf)
+
+## Development Roadmap 
+
+### Overview
+
+- **Total Estimated Duration:** ~3-6 months
+- **Full-Time Equivalent (FTE):**  1,4 FTE
+- **Total Costs:** 3 BTC 
+
+### Milestone 1 Example — Basic functionality
```

Comment:
Please replace template text

Reply:
Replaced template text in this [commit](https://github.com/OpenSmartContract/W3F-Program/commit/16385bba4c45b5ba0dcd246542d4ba586e5e93c6)